### PR TITLE
DT-2404: 💄 Switch to sign in / out instead of login / out

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ services:
       test: ["CMD", "curl", "-f", "http://localhost:8080/auth/health"]
     environment:
       - SPRING_PROFILES_ACTIVE=dev
-      - DELIUS_ENABLED=false
```
Comment (petergphillips) [inline docker-compose.yml:25]: `dev` profile disables delius anyway

File: docker-compose.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ services:
...
+      - APPLICATION_AUTHENTICATION_UI_ALLOWLIST=0.0.0.0/0
```
Comment (petergphillips) [inline docker-compose.yml:25]: allows access to the client configuration within auth

File: .husky/.gitignore [hunk 3]
```diff
@@ -1 +0,0 @@
-_
```
Comment (petergphillips) [inline .husky/.gitignore:1]: removed as not needed in latest version of husky

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**